cli/command/service/create.go

```diff
@@ -40,6 +40,7 @@ func newCreateCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags.StringSliceVar(&opts.networks, flagNetwork, []string{}, "Network attachments")
 	flags.VarP(&opts.endpoint.ports, flagPublish, "p", "Publish a port as a node port")
 	flags.StringSliceVar(&opts.groups, flagGroup, []string{}, "Set one or more supplementary user groups for the container")
+	flags.Var(&opts.extraHosts, flagAddHost, "Add a custom host-to-IP mapping (host:ip)")
```

Comment:
To follow convention, this should be:

`--host`

Then, `docker service update` should expose:
- `--host-add`
- `--host-rm`

@dnephin Thoughts?